### PR TITLE
[fix: map] title does not change when per capita option is selected

### PR DESCRIPTION
**Pull request by sharadshriram**
This PR attempts to resolve issue #1357 

In my local tests, I am able to see the desired chart title captured in the following screenshots

## Before the proposed change
image

### After review comments incorporation
image

### Before review
image

### CODE REVIEW

File: static/js/tools/map/chart.tsx [hunk 1]
```diff
@@ -89,10 +89,11 @@ export function Chart(props: ChartProps): JSX.Element {
   const statVarInfo = props.statVar.value;
   const [errorMessage, setErrorMessage] = useState("");
   const [denomInput, setDenomInput] = useState(props.statVar.value.denom);
-  const title = getTitle(
+  const chart_title=   getTitle(
     Array.from(props.dates),
     statVarInfo.info.title ? statVarInfo.info.title : statVarInfo.dcid
   );
+  const title = props.statVar.value.perCapita? 'Per Capita ' + chart_title : chart_title;
```
Comment (chejennifer) [inline static/js/tools/map/chart.tsx:96]: In Scatter Tool, we append "Per Capita" to the end of the stat var name. To keep things consistent, let's move "Per Capita" to the end of the chart title?

**Review by chejennifer — COMMENTED** [review]
Thanks for the fix!

File: static/js/tools/map/chart.tsx (hunk 1 above)
Comment (beets) [inline static/js/tools/map/chart.tsx:96]: please follow the convention from scatter (per capita comes after the sv name)

this edit also belongs to the getTitle function, passing in props.statVar.value.perCapita

also, as an fyi - we use camelcase in typescript

**Review by beets — COMMENTED** [review]
thanks for the bug fix!

File: static/js/tools/map/chart.tsx (hunk 1 above)
Comment (sharadshriram) [reply static/js/tools/map/chart.tsx:97]: Thank you for the review. I have moved 'Per Capita' to the end of the StatVar's name.

Comment (sharadshriram) [reply static/js/tools/map/chart.tsx:97]: Thank you for the review. I have moved 'Per Capita' to the end of the StatVar's name with the edit being made in the `getTitle` function.

Comment (chejennifer) [reply static/js/tools/map/chart.tsx:97]: With the edit in the `getTitle` function, line 97 should be removed. And maybe change `chart_title` back to `title`?

Comment (sharadshriram) [reply static/js/tools/map/chart.tsx]: Thanks for the review, Jennifer. I have cleaned up the code. This particular issue when I tried resolving a git divergent branches error. :)

**Comment by chejennifer** [comment]
Thanks for the changes! There are some linting issues causing tests to fail. Running `./run_test -f` should fix those issues!

**Review by chejennifer — APPROVED** [review]

**Pull request merged**